bin/gobuild.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script builds and link stamps the output
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+OUT=${1:?"output path"}
+VERSION_PACKAGE=${2:?"version go package"} # istio.io/istio/mixer/pkg/version
+BUILDPATH=${3:?"path to build"}
+
+set -ex
+
+VERBOSE=${VERBOSE:-"0"}
+V=""
+if [[ "${VERBOSE}" == "1" ]];then
+    V="-x"
+fi
+
+GOOS=${GOOS:-linux}
+GOARCH=${GOARCH:-amd64}
+GOBIN=${GOBIN:-go}
+BUILDINFO=${BUILDINFO:-""}
+STATIC=${STATIC:-1}
+CGO="CGO_ENABLED=0"
+LDFLAGS="-extldflags -static"
+
+if [[ "${STATIC}" !=  "1" ]];then
```

Comment:
Use STATIC values consistently, there's 0 passed around, but 1/-1 here.

Reply:
static is the default. You can pass `STATIC=0` to disable static compilation.